commit d5c0f12c8fee270132142c5d19ec5820a051dd80 (refs/changes/12/3612/3)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-03-19T15:41:06-04:00 (1 year, 7 months ago)
    
    TRITON-240 zoneevent should be called with an identifier string
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/lib/backends/smartos/index.js b/lib/backends/smartos/index.js
index beae4fc..7df68b7 100644
--- a/lib/backends/smartos/index.js
+++ b/lib/backends/smartos/index.js
@@ -505,7 +505,8 @@ function startZoneeventWatcher(opts, watchers) {
     assert.object(opts, 'opts');
     assert.object(opts.log, 'opts.log');
 
-    watchers.zoneeventHandle = spawn('/usr/vm/sbin/zoneevent', []);
+    watchers.zoneeventHandle = spawn('/usr/vm/sbin/zoneevent',
+        ['-i', 'cn-agent']);
     pid = watchers.zoneeventHandle.pid;
     opts.log.info('zoneevent[' + pid + '] watcher running');
 
